src/api/app/views/webui/users/tokens/show.html.haml

```diff
@@ -0,0 +1,64 @@
+- @pagetitle = 'Show Token'
+
+.card
+  .card-body
+    %h3.mb-3= @pagetitle
+
+    .row
+      .col-12.col-md-10.col-lg-8
+        %p.font-weight-bold Your new OBS Personal Access Token
```

Comment:
This text should be rendered only right after the creation of the token, so when we show the token string with `session[:show_token]`. Afterwards, the token isn't new anymore.